src/crypto/OlmDevice.ts

```diff
@@ -1119,7 +1119,7 @@ export class OlmDevice {
                                 }
                             }
 
-                            logger.info(
+                            logger.debug(
```

Comment:
Why is this not needed in the rageshakes? I'm not one to debug crypto issues, but that somewhat looks relevant to them?

Reply:
This doesn't affect whether or not it goes into rageshakes. It *only* affects whether or not a default console configuration will show it or not.